### PR TITLE
README.md add standardx

### PR DESCRIPTION
**Pull request by brodybits**
I just discovered standardx from <https://github.com/standard/standard/issues/400#issuecomment-412537643>, please include!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -798,6 +798,7 @@ There are also many **[editor plugins](#are-there-text-editor-plugins)**, a list
 **[npm packages that use `standard`](https://github.com/standard/standard-packages)**,
 and an awesome list of
 **[packages in the `standard` ecosystem](https://github.com/standard/awesome-standard)**.
+- **[standardx](https://github.com/standard/standardx)** - standard, with custom tweaks
```
Comment (LinusU) [inline README.md]: Can you add this in conjunction with the list? Basically move this line 5 lines up, so it's right after `semistandard`.

**Comment by brodybits** [comment]
Just moved the entry as requested, looking forward to the merge:)

**Comment by LinusU** [comment]
failing because of https://github.com/eddiemoore/gulp-codecov/pull/69, merging anyway

**Comment by brodybits** [comment]
Thanks @LinusU! I already raised alternative PRs to resolve the build failure and cleanup the duplicate entry:
- standard/standard-packages#22 - cleanup duplicate & disable failing gulp-codecov
- standard/standard-packages#23 - cleanup duplicate gulp-codecov entry

**Pull request merged**